README.md

```diff
@@ -255,6 +255,12 @@ tmux kill-server & tmux
 set -g @catppuccin_status_left_separator ""
 ```
 
+#### Set the status module middle separator:
+
+```sh
+set -g @catppuccin_status_middle_separator ""
```

Comment:
```suggestion
set -g @catppuccin_status_middle_separator "null"
```
Setting it to `""` causes it to fallback to the default (which is in this case happens to be `""`).

Reply:
Got it, thank you for the catch.